includes/Gateway/Cash_App_Pay_Gateway.php

```diff
@@ -0,0 +1,1226 @@
+					'</a>'
+				),
+				'wc-square-enable-cash-app-pay',
+			);
+		}
+	}
+
+	/**
+	 * Get the default payment method title, which is configurable within the
+	 * admin and displayed on checkout
+	 *
+	 * @since x.x.x
+	 * @return string payment method title to show on checkout
+	 */
+	protected function get_default_title() {
+		return esc_html__( 'Cash App Pay', 'woocommerce-square' );
+	}
+
+
+	/**
+	 * Get the default payment method description, which is configurable
+	 * within the admin and displayed on checkout
+	 *
+	 * @since x.x.x
+	 * @return string payment method description to show on checkout
+	 */
+	protected function get_default_description() {
+		return esc_html__( 'Pay securely using Cash App Pay.', 'woocommerce-square' );
+	}
+
+	/**
+	 * Get transaction URL format.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string URL format
+	 */
+	public function get_transaction_url_format() {
+		return $this->get_plugin()->get_settings_handler()->is_sandbox() ? 'https://squareupsandbox.com/dashboard/sales/transactions/%s' : 'https://squareup.com/dashboard/sales/transactions/%s';
+	}
+
+	/**
+	 * Initialize payment gateway settings fields
+	 *
+	 * @since x.x.x
+	 * @see WC_Settings_API::init_form_fields()
+	 */
+	public function init_form_fields() {
+
+		// common top form fields
+		$this->form_fields = array(
+			'enabled'      => array(
+				'title'   => esc_html__( 'Enable / Disable', 'woocommerce-square' ),
+				'label'   => esc_html__( 'Enable this gateway', 'woocommerce-square' ),
+				'type'    => 'checkbox',
+				'default' => 'no',
+			),
+
+			'title'        => array(
+				'title'    => esc_html__( 'Title', 'woocommerce-square' ),
+				'type'     => 'text',
+				'desc_tip' => esc_html__( 'Payment method title that the customer will see during checkout.', 'woocommerce-square' ),
+				'default'  => $this->get_default_title(),
+			),
+
+			'description'  => array(
+				'title'    => esc_html__( 'Description', 'woocommerce-square' ),
+				'type'     => 'textarea',
+				'desc_tip' => esc_html__( 'Payment method description that the customer will see during checkout.', 'woocommerce-square' ),
+				'default'  => $this->get_default_description(),
+			),
+
+			'button_theme' => array(
+				'title'    => esc_html__( 'Cash App Pay Button Theme', 'woocommerce-square' ),
+				'desc_tip' => esc_html__( 'Select the theme of the Cash App Pay button.', 'woocommerce-square' ),
+				'type'     => 'select',
+				'default'  => 'dark',
+				'class'    => 'wc-enhanced-select wc-square-cash-app-pay-options',
+				'options'  => array(
+					'dark'  => esc_html__( 'Dark', 'woocommerce-square' ),
+					'light' => esc_html__( 'Light', 'woocommerce-square' ),
+				),
+			),
+
+			'button_shape' => array(
+				'title'    => esc_html__( 'Cash App Pay Button shape', 'woocommerce-square' ),
+				'desc_tip' => esc_html__( 'Select the shape of the Cash App Pay button.', 'woocommerce-square' ),
+				'type'     => 'select',
+				'default'  => 'semiround',
```

Comment:
Does this match the same style set in our Express Payment Button PT work so that we visually try and default into the same button styles?

Reply:
Yes, this default semiround button style is the same as we have in the Express Payment Button PT, and it also matches the default square style that the test site has.

![image](https://github.com/woocommerce/woocommerce-square/assets/10613171/5e8da779-ac6c-4f1b-9069-e8a02034cfba)